packages/twenty-front/src/modules/object-record/record-show/hooks/useRecordShowPagePagination.ts

```diff
@@ -147,6 +148,19 @@ export const useRecordShowPagePagination = (
 
   const objectLabel = capitalize(objectMetadataItem.namePlural);
 
+  let totalCount = 0;
+
+  if (totalCountBefore !== 0 && totalCountAfter !== 0) {
+    totalCount = Math.max(totalCountBefore, totalCountAfter);
+  } else {
+    totalCount =
```

Comment:
totalCount = (totalCountBefore === 0 && totalCountAfter === 0) ? 1 : Math.max(totalCountBefore, totalCountAfter)
I believe this should work?

Reply:
Yup. That's a better way to write it. Let me correct this.